sdk/device_test.go

```diff
@@ -447,7 +447,63 @@ func TestNewDeviceFromConfig9(t *testing.T) {
 }
 
 func TestNewDeviceFromConfig10(t *testing.T) {
-	// Unknown transformation function specified
+	// Proto and instance both define transformers - ensure they merge correctly.
+	proto := &config.DeviceProto{
+		Type: "type1",
+		Data: map[string]interface{}{
+			"port": 5000,
+		},
+		Context: map[string]string{
+			"foo": "bar",
+		},
+		Tags:         []string{"default/foo"},
+		Handler:      "testhandler",
+		WriteTimeout: 3 * time.Second,
+		Transforms: []*config.TransformConfig{
+			{Apply: "FtoC"},
+			{Scale: "3"},
```

Comment:
The case of unit conversion prior to scale will fail, maybe working by accident in corner cases. The scale has to be done first. It's fine that the order of operations can be reversed, it's just not going to work.

Reply:
Yeah, for our use case, thats right. This test is just to ensure that the order of the transforms are honored -- not necessarily that the transformations themselves make sense here.